### DISCUSSION TITLE
Welcome to Java-Questions-and-Solutions Discussions!

### DISCUSSION THREAD

**Discussion by SarthakKeshari** [SarthakKeshari/Java-Questions-and-Solutions#2] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ln11211** [comment]
do issues in this repo come under hacktoberfest ?

**Reply by SarthakKeshari** [reply]
Yes, Hacktoberfest tags and approvals will be up from 1st October.

**Reply by Ln11211** [reply]
@SarthakKeshari sir, I was assigned a issue yesterday and the deadline is tomorrow, I have completed working on the issue, can I wait and submit a pull request at october 1 or before(ie, my deadline of 30/9) ?

**Reply by SarthakKeshari** [reply]
@Ln11211, Kindly let me know issue id

**Reply by Ln11211** [reply]
@SarthakKeshari sir, it was Birthday Cake Candles #5
